Add README file and simplify Makefile

davidcassany opened the pull request:
This commit provides a simple README file simplifies the Makefile since
the purpose of this project is no providing tones of options but
providing simple skeleton for anyone to get started tweaking.

davidcassany committed:
Add README file and simplify Makefile

This commit provides a simple README file simplifies the Makefile since
the purpose of this project is no providing tones of options but
providing simple skeleton for anyone to get started tweaking.

Signed-off-by: David Cassany <dcassany@suse.com>

mudler commented on Makefile:
```diff
@@ -41,33 +40,26 @@ endif
 	--values $(ROOT_DIR)/packages/cOS/values/$(FLAVOR).yaml \
 	--tree=$(TREE) $(PACKAGES) \
-	--destination $(DESTINATION)
+	--destination $(ROOT_DIR)/build
```
Maybe we can omit this completely, IIRC it is implied $PWD/build if not specified

mudler commented on Makefile:
```diff
@@ -41,33 +40,26 @@ endif
 	$(LUET) create-repo $(PUBLISH_ARGS) --tree "$(TREE)" \
     --output $(FINAL_REPO) \
-    --packages $(DESTINATION) \
+    --packages $(ROOT_DIR)/build \
```
Same :arrow_up:  here

mudler commented on README.md:
```diff
@@ -0,0 +1,91 @@
+# SampleOS
+
+SampleOS is a dummy example of a derivate system based on containerOS (**cOS**).
```
```suggestion
SampleOS is a dummy example of a derivate system built with containerOS-toolkit (**cOS-toolkit**).
```

mudler commented on README.md:
````diff
@@ -0,0 +1,91 @@
+$ qemu-kvm -m 2048 -cdrom sampleOS-0.20210415.iso -hda test.img
+```
+
+After loging into the VM you can check SampleService by typing:
````
Maybe we should mention default username and password here

mudler commented on README.md:
```diff
@@ -0,0 +1,91 @@
+
+## Build SampleOS Locally
+
+SampleOS can be build locally by using the following make targets.
```
Can we also port here the .envrc from cOS? it might be even easier for newcomers

mudler commented:
Just few nitpicks, but otherwise looks good here :+1:

davidcassany committed:
Update README.md

Co-authored-by: Ettore Di Giacinto <mudler@users.noreply.github.com>
Signed-off-by: David Cassany <dcassany@suse.com>

davidcassany committed:
Remove .gitmodules and further simplify Makefile

This commit removes the git submodule, simplifies the Makefile and
includes the containerized build environment.

Signed-off-by: David Cassany <dcassany@suse.com>
